lib/child_process.js

```diff
@@ -633,17 +623,16 @@ function execFileSync(/*command, args, options*/) {
 }
 exports.execFileSync = execFileSync;
 
+function execSync(command, rawArgs, rawOpts) {
```

Comment:
This accepts only two arguments, no?

Reply:
Yes, and I just realized that args was always undefined here. Interesting ;-)